firewood/src/db.rs

```diff
@@ -609,9 +593,6 @@ impl Db {
         // recover from Wal
         disk_requester.init_wal("wal", &db_path);
 
-        let root_hash_staging = StoreRevMut::new(root_hash_cache);
-        let reset_headers = reset;
```

Comment:
Removed -- we do `reset_headers: reset` below instead. Similar for `root_hash_staging`